### DISCUSSION TITLE
Using Figma variables

### DISCUSSION THREAD

**Discussion by ayntwan** [primefaces/community#149] [Figma UI Kit] [answered]
Hi, 

It doesn't seem to me that the UI kit makes use of Figma's new features, in particular the variables. 
Will there be an update? 

At the moment, if I want to change the primary color, do I have to change it on all the components that use it? 

Thanks in advance for your answers

**Comment by davidsee123** [comment]
I also wonder the same question

**Comment by onursenture** [comment, marked as answer]
Hello,

We're continuing our work on the new theming engine for our UI libraries; parallel to this, we are also working on Tokens for our Figma UI Kit. We decided to work with the Tokens Studio plugin for Tokens because Figma's native solution, Variables, is not a full-fledged Design Tokens tool.

Our developments with Tokens Studio started with version 2.1.0. The progress of the Tokens work, which changes all components by changing the Primary color, needs to progress along with the theming engine work. Ultimately, our goal is to provide a kit that will save time for both designers and developers. We are working hard to achieve this. I appreciate your patience and good luck with your work.

We recommend you watch this video: https://www.youtube.com/watch?v=AtMVxP0UMII
